src/org/myworldgis/projection/Ellipsoid.java

```diff
@@ -164,7 +164,7 @@
     public final static Ellipsoid WGS_72 = new Ellipsoid(true, "WGS 72", 6378135.0, SI.METRE, 0.006694318); 
     
     /** */
-    public final static Ellipsoid WGS_84 = new Ellipsoid(true, "WGS 84", 6378137.0, SI.METRE, 0.00669438); 
+    public final static Ellipsoid WGS_84 = new Ellipsoid(true, "WGS 84", 6378137.0, SI.METRE, 0.0066943799901413165); 
```

Comment:
I'd recommend adding a comment here about why the precision in the final value is necessary.